tests/test_series.py

```diff
@@ -999,6 +999,12 @@ def test_types_getitem() -> None:
     s3: pd.Series = s[:2]
 
 
+def test_types_getitem_by_timestamp() -> None:
+    index = pd.date_range("2018-01-01", periods=2, freq="D")
+    series = pd.Series(range(2), index=index)
+    check(assert_type(series[index[-1]], int), np.integer)
```

Comment:
I know, it was me initially who added a check for the return type but I see it useless now.

`mypy`/`pyright` will check if it's legal:

```suggestion
    series[index[-1]]
```

Reply:
I wouldn't mind keeping the return check, even if it isn't necessary for your change

Reply:
I agree.  We're hoping to get all the tests to use that pattern.